config/index.js

```diff
@@ -81,11 +85,19 @@ module.exports = {
       logout_url: _authUrl(
         `/auth/logout?client_id=${AUTH_KEY}&redirect_uri=${BASE_URL}`
       ),
+      groups_url: userName => {
+        return _authUrl(`/auth/api/authuser/${userName}/groups`)
+      },
+      base_url: AUTH_BASE_URL,
```

Comment:
Do we need to add this? It looks like it's being used by the tests and pulled in direct to create the mock?

I think if we stub the url in the test using `proxyquire` it prevents us needing to add this to the config.

Reply:
Good shout.